ruby/ql/lib/codeql/ruby/security/UnsafeCodeConstructionCustomizations.qll

```diff
@@ -96,6 +96,25 @@ module UnsafeCodeConstruction {
     override string getSinkType() { result = "string interpolation" }
   }
 
+  /**
+   * A string constructed from a string-concatenation (e.g. `"foo " + sink`),
```

Comment:
Nitpick: "constructed from" initially made me think that this would be the node corresponding to the whole `AddExprRoot` rather than the relevant leaf node of that `AddExprRoot`. Maybe something like:
```suggestion
   * A component of a string-concatenation (e.g. `"foo " + sink`),
```